docs/components/bottom_sheets.md

```diff
@@ -0,0 +1,280 @@
+            return Center(
+              child: FlatButton(
+                child: Text('Show Bottom Sheet'),
+                onPressed: () {
+                  showBottomSheet(
+                    context: context,
+                    builder: (context) {
+                      final theme = Theme.of(context);
+                      // Using Wrap makes the bottom sheet height the height of the content.
+                      // Otherwise, the height will be half the height of the screen.
+                      return Wrap(
+                        children: [
+                          ListTile(
+                            title: Text(
+                              'Header',
+                              style: theme.textTheme.subtitle1
+                                  .copyWith(color: theme.colorScheme.onPrimary),
+                            ),
+                            tileColor: theme.colorScheme.primary,
+                          ),
+                          ListTile(
+                            title: Text('Title 1'),
+                          ),
+                          ListTile(
+                            title: Text('Title 2'),
+                          ),
+                          ListTile(
+                            title: Text('Title 3'),
+                          ),
+                          ListTile(
+                            title: Text('Title 4'),
+                          ),
+                          ListTile(
+                            title: Text('Title 5'),
+                          ),
+                        ],
+                      );
+                    },
+                  );
+                },
+              ),
+            );
+          },
+        ),
+      ),
+    );
+  }
+}
+```
+
+The persistent bottom sheet can be used for a standard bottom sheet.
+Use a [DraggableScrollableSheet](https://api.flutter.dev/flutter/widgets/DraggableScrollableSheet-class.html) for more custom dragging and snap points.
+
+### Anatomy and key properties
+
+The following shows the anatomy of a standard bottom sheet:
+
+![Standard bottom sheet anatomy](assets/bottom_sheets/standard-anatomy.png)
+
+1. Sheet
+1. Contents
+
+#### Sheets properties
+
+&nbsp;         | Property
+-------------- | ------------------------
+| **Color** | `backgroundColor` |
+| **Elevation** | `elevation` |
+| **Shape** | `shape` |
+
+#### Contents properties
+
+There are no specific properties for content because the content can be any composition of widgets.
+
+## Modal bottom sheet
+
+Modal bottom sheets present a set of choices while blocking interaction with the rest of the screen. They are an alternative to inline menus and simple dialogs on mobile, providing additional room for content, iconography, and actions.
+
+Modal bottom sheets are used in mobile apps only.
+
+### Modal bottom sheet example
+
+`BottomSheet`
+- [Class definition](https://api.flutter.dev/flutter/material/BottomSheet-class.html)
```

Comment:
This should probably link to the showModalBottomSheet helper instead

Reply:
Good idea